helper/common_handler.js

```diff
@@ -3116,6 +3116,274 @@ async function ganttChartObject(data) {
+
+                                                                                i = i + 1;
+
+                                                                            }
+                                                                        });
+                                                                        rimraf(imgPath, function () { console.log("done"); });
+
+                                                                    } catch (ex) {
+                                                                        console.log("ex ", ex);
+                                                                    }
+
+                                                                    let response = {
+                                                                        status: "success",
+                                                                        message: 'report generated',
+                                                                        pdfUrl: signedRes,
+                                                                        downloadPath: data.key
+                                                                    };
+                                                                    resolve(response);
+                                                                })
+                                                            });
+
+                                                        }
+
+                                                    });
+                                                }
+                                                uploadFile();
+                                            }
+                                        });
+                                    }
+
+                                }).catch(err => {
+                                    resolve(err);
+                                })
+                        }
+                    })
+                })
+        }
+        catch (err) {
+            resolve(err);
+        }
+    })
+}
+
+async function getEntityReportChartObjects(data) {
+
+    return new Promise(async function (resolve, reject) {
+
+        let chartData = [];
+
+        let getChartObjects = [
+            getTaskOverviewChart(data.tasks),
+            getCategoryWiseChart(data.categories)
+        ];
+
+        await Promise.all(getChartObjects)
+            .then(function (response) {
+                chartData.push(response[0]);
+                chartData.push(response[1]);
+            });
+
+        return resolve(chartData)
+
+       
+    })
+}
+
+async function getTaskOverviewChart(tasks) {
+    return new Promise( async function(resolve,reject) {
+
+         let seriesData="[";
+         Object.keys(tasks).forEach( eachTask => {
+            let color = "";
+            if(eachTask=="Completed"){
+                color ="green";
+            }else if(eachTask=="Not Started"){
```

Comment:
@rakeshSgr - Is there a space in status?

Reply:
Yes, here we will get status with title case only, as we need to show status like this in pdf